src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -95,11 +107,15 @@ def register_path_argument(scope, default_file_param=None, options_list=None):
 directory_type = CliArgumentType(options_list=('--directory-name', '-d'), help='The directory name.', completer=get_storage_name_completion_list(FileService, 'list_directories_and_files', parent='share_name'))
 file_name_type = CliArgumentType(options_list=('--file-name', '-f'), completer=get_storage_name_completion_list(FileService, 'list_directories_and_files', parent='share_name'))
 share_name_type = CliArgumentType(options_list=('--share-name', '-s'), help='The file share name.', completer=get_storage_name_completion_list(FileService, 'list_shares'))
+table_name_type = CliArgumentType(options_list=('--table-name', '-t'), completer=get_storage_name_completion_list(TableService, 'list_tables'))
+queue_name_type = CliArgumentType(options_list=('--queue-name', '-q'), help='The queue name.', completer=get_storage_name_completion_list(QueueService, 'list_queues'))
+IGNORE_TYPE = CliArgumentType(help=argparse.SUPPRESS, nargs='?', action=IgnoreAction, required=False)
 
```

Comment:
Consider moving the definition for IGNORE_TYPE higher up, e.g. before any function definitions, as it is a constant. It is used by register_path_argument but the actual definition is below it which doesn't feel quite right.

Also, is the intention to make this available outside storage module as well as a piece of core functionality? [https://github.com/Azure/azure-cli/issues/675#issuecomment-239030028](https://github.com/Azure/azure-cli/issues/675#issuecomment-239030028)